packages/drivers/debugger/src/fluidDebuggerController.ts

```diff
@@ -144,6 +149,24 @@ export class DebugReplayController extends ReplayController implements IDebugger
         reader.readAsText(file, "utf-8");
     }
 
+    public async onDownloadOps(): Promise<string> {
+        if (this.documentService !== undefined) {
+            const documentDeltaStorageService = await this.documentService.connectToDeltaStorage();
+            const messages = await this.fetchOpsFromDeltaStorage(documentDeltaStorageService);
+            return JSON.stringify(messages, undefined, 2);
+        }
+        return "N/A";
```

Comment:
should this be an exception? I.e. when do we get here?

Reply:
Would be if we hit this before initStorage somehow.  An exception probably makes sense

Reply:
Ideal solution is that it waits for initStorage to finish or at least finish the part that it requires. Or disable a button until it's done.

Reply:
+1 for throw here

Reply:
Agree on the ideal solution (disable or wait). Changed it to throw for now.